internal/cmd/keys.go

```diff
@@ -150,7 +168,26 @@ func newKeysRemoveCmd(cli *CLI) *cobra.Command {
 				return err
 			}
 
-			err = client.DeleteAccessKeyByName(ctx, args[0])
+			config, err := currentHostConfig()
+			if err != nil {
+				return err
+			}
+
+			keyName := args[0]
+
+			userID := config.UserID
+			keys, err := listAll(ctx, client.ListAccessKeys, api.ListAccessKeysRequest{Name: keyName, UserID: userID})
```

Comment:
Is `infra keys remove` losing a way for admins to remove access keys of other users? Previously an admin could list all the keys and remove any keys in their org. It looks like the remove part is no longer possible. 

I suspect we don't want to lose that.

Now that key names are not unique across the org, I guess we need a `--user` flag here so the admin can select the user that owns the key.

Reply:
Good catch. The `--user` flag should be here too.